src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -31,7 +31,7 @@ public class RazorViewEngineOptions
         /// locate Razor files.
         /// </summary>
         /// <remarks>
-        /// At startup, this is initialized to include an instance of <see cref="PhysicalFileProvider"/> that is
+        /// At startup, this is initialized to include an instance of PhysicalFileProvider that is
```

Comment:
A breaking change because this project no longer has a transitive reference to `Microsoft.Extensions.FileProviders.Physical`. Up to us if that's ok. If we decide to keep the change we should change this documentation to not reference the `PhysicalFileProvider` directly.